blocks/gallery-block/features/gallery/default.jsx

```diff
@@ -1,35 +1,65 @@
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
 import PropTypes from 'prop-types';
-import { useFusionContext } from 'fusion:context';
+import { useContent } from 'fusion:content';
+import { useFusionContext, useAppContext } from 'fusion:context';
 import getProperties from 'fusion:properties';
 import getTranslatedPhrases from 'fusion:intl';
-import GlobalContentGallery from './_children/global-content';
-import CustomContentGallery from './_children/custom-content';
+
+import { Gallery, ErrorBoundary } from '@wpmedia/engine-theme-sdk';
+
+const AdFeature = lazy(/* istanbul ignore next */ () => import('@wpmedia/ads-block'));
```

Comment:
```suggestion
const AdFeature = lazy(() => import('@wpmedia/ads-block'));
```

can we remove istanbul ignore. don't belive we're actually using this

Reply:
As this is an anonymous function you can not get test coverage to cover this - add this comment makes the coverage report ignore it as part of the %'s.
Istanbul is the reporter tool that Jest uses under the covers